bufconn_test.go

```diff
@@ -26,12 +27,13 @@ func TestRead(t *testing.T) {
 	if !assert.Equal(t, alphabet+"\n", line1) {
 		return
 	}
+	response.WriteString(alphabet)
```

Comment:
To make sure that data written afterwards can still be read.

Reply:
Nice improvement to the test.